blockdevice/blockdevice.go

```diff
@@ -282,6 +309,12 @@ type TemperatureInformation struct {
 
 	// CurrentTemperature is the temperature of the drive in celsius
 	CurrentTemperature int16
+
+	// LowestTemperature is the temperature of the drive in celsius
+	LowestTemperature int16
+
+	// HighestTemperature is the temperature of the drive in celsius
```

Comment:
Comments should describe what the fields are and what information they convey. Since this is the only place where all blockdevice information is stored, the comments can be pretty descriptive of what the field is.